buildenv/docker/mkdocker.sh

```diff
@@ -372,6 +372,7 @@ fi
   echo "    perl-devel \\"
   echo "    perl-ExtUtils-MakeMaker \\" # required by git
   echo "    perl-GD \\"
+  echo "    perl-IPC-Cmd \\"
```

Comment:
Please explain why this is needed and what it helps. Is it needed only for centos:6?

Reply:
We can't compile openssl v3 without this, see infrastructure/issues/6790 for more details.

Reply:
It's needed for all OSes where builds occur. OpenJ9 only builds xlinux on centos:6.

Reply:
It would have helped if that fact were mentioned here.

Reply:
Comment descriptor added in latest push.